MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -769,7 +770,7 @@
 			}
 			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_TEXEL_BUFFER_ALIGNMENT_FEATURES_EXT: {
 				auto* texelBuffAlignFeatures = (VkPhysicalDeviceTexelBufferAlignmentFeaturesEXT*)next;
-				texelBuffAlignFeatures->texelBufferAlignment = _metalFeatures.texelBuffers && [_mtlDevice respondsToSelector: @selector(minimumLinearTextureAlignmentForPixelFormat:)];
+				texelBuffAlignFeatures->texelBufferAlignment = [_mtlDevice respondsToSelector: @selector(minimumLinearTextureAlignmentForPixelFormat:)];
```

Comment:
That method is available starting in macOS 10.13 and iOS 11. We could probably get away with:
```suggestion
				texelBuffAlignFeatures->texelBufferAlignment = true;
```